doc/api/buffer.md

```diff
@@ -188,6 +197,10 @@ that the server actually returned win-1252-encoded data, and using `'latin1'`
 encoding may incorrectly decode the characters.
 
 ## Buffers and TypedArray
+changes:
+  - version: v3.0.0
+    pr-url: https://github.com/nodejs/node/pull/2002
+    description: The `Buffer`s class now inherits from `Uint8Array`.
```

Comment:
Should this be in `<!-- YAML ... -->` markers or do I misunderstand how it works?